api/src/main/java/io/smallrye/dux/spi/ServiceDiscoveryProvider.java

```diff
@@ -0,0 +1,10 @@
+package io.smallrye.dux.spi;
```

Comment:
What means `spi`?

Reply:
spi is for people who implement new service discovery and load balancer strategies. It stands for Service Provider Interface